geniza/corpus/models.py

```diff
@@ -459,21 +459,6 @@ def shelfmark(self):
             )
         )
 
-    @property
-    def shelfmark_display(self):
-        """First shelfmark plus join indicator for shorter display."""
-        # NOTE preliminary pending more discussion and implementation of #154:
-        # https://github.com/Princeton-CDH/geniza/issues/154
-        certain = list(
-            dict.fromkeys(
-                block.fragment.shelfmark
-                for block in self.textblock_set.filter(certain=True)
-            ).keys()
-        )
-        if not certain:
-            return None
-        return certain[0] + (" + …" if len(certain) > 1 else "")
-
     @property
```

Comment:
Are we not using this anywhere anymore?

I am hesitant to remove it entirely because I suspect we'll still end up using some version of this (maybe with a minimum number of shelfmarks that is higher than 1). Would prefer to keep the method and test with a note in the comments that it isn't currently used and should either be revised or removed if we don't end up using it.

Reply:
Done in 15fde44 👍